src/account-settings/data/constants.js

```diff
@@ -1,8 +1,9 @@
 export const YEAR_OF_BIRTH_OPTIONS = (() => {
   const currentYear = new Date().getFullYear();
   const years = [];
+  // Create a list of possible birth years, limiting the minimum user age to 16 years
   let startYear = currentYear - 120;
-  while (startYear < currentYear) {
+  while (startYear < currentYear - 16) {
```

Comment:
I would create a separate constant variable for `16`

Reply:
done